common/src/test/java/org/opensearch/ml/common/transport/connector/MLConnectorGetResponseTests.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.ml.common.transport.connector;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.opensearch.action.ActionResponse;
+import org.opensearch.common.Strings;
+import org.opensearch.common.io.stream.BytesStreamOutput;
+import org.opensearch.common.io.stream.StreamOutput;
+import org.opensearch.common.xcontent.XContentFactory;
+import org.opensearch.common.xcontent.XContentType;
+import org.opensearch.core.xcontent.ToXContent;
+import org.opensearch.core.xcontent.XContentBuilder;
+import org.opensearch.ml.common.connector.Connector;
+import org.opensearch.ml.common.connector.HttpConnectorTest;
+
+import java.io.IOException;
+import java.io.UncheckedIOException;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNotSame;
+import static org.junit.Assert.assertSame;
+
+public class MLConnectorGetResponseTests {
+    Connector connector;
+
+    @Before
+    public void setUp() {
+        connector = HttpConnectorTest.createHttpConnector();
+    }
+
+    @Test
+    public void writeTo_Success() throws IOException {
+        BytesStreamOutput bytesStreamOutput = new BytesStreamOutput();
+        MLConnectorGetResponse response = MLConnectorGetResponse.builder().mlConnector(connector).build();
+        response.writeTo(bytesStreamOutput);
+        MLConnectorGetResponse parsedResponse = new MLConnectorGetResponse(bytesStreamOutput.bytes().streamInput());
+        assertNotEquals(response, parsedResponse);
+        assertNotSame(response.mlConnector, parsedResponse.mlConnector);
+        assertEquals(response.mlConnector, parsedResponse.mlConnector);
```

Comment:
This class behaves different from https://github.com/opensearch-project/ml-commons/blob/2.x/common/src/test/java/org/opensearch/ml/common/transport/model/MLModelGetResponseTest.java#L48, is this expected? @ylwu-amzn